xmake.lua

```diff
@@ -0,0 +1,32 @@
+set_config("ue4ssRoot", os.curdir())
+set_config("scriptsRoot", path.join(os.curdir(), "tools/xmakescripts"))
+
```

Comment:
QoL, consider using `set_group("deps")` to allow for grouping of projects when VS project is generated/updated. This would help with the organization of the generated solution.
<img width="470" alt="PrettyStuff" src="https://github.com/UE4SS-RE/RE-UE4SS/assets/7477000/a5d18037-f459-4b7c-8a78-cdc16df25385">

xmake also supports building specific groups so it could be interesting to have a `set_group("mods")` in the boilerplate mod `xmake.lua` to allow for someone to work on multiple mods and build them with `xmake -g mods`. Something to consider...